app/assets/stylesheets/overlays/_activity-feed-overlay.scss

```diff
@@ -4,8 +4,7 @@
   img { margin-right: 6px; }
 
   table {
-    max-width: 900px;
-    min-width: 550px;
+    width: 700px;;
```

Comment:
Properties should be ordered margin, width<br>Declaration should be terminated by a single semicolon